src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/data/GraphQLEntrypoint.java

```diff
@@ -0,0 +1,25 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data;
+
+import java.util.Map;
+
+/**
+ * Used to represent and contain information about a specific field
+ * in the GraphQL query type
+ */
+public interface GraphQLEntrypoint {
+    /**
+     * @return the GraphQL query fieldname
+     */
+    public String getFieldName();
+
+    /**
+     * @return the parameter definitions for the entrypoint as a map
+     * consisting of parameter names mapped to parameter value types.
+     */
+    public Map<String,String> getParameterDefinitions();
```

Comment:
Is this meant to be about the arguments defined for a corresponding field? If so, please use the term "argument" instead of "parameter" to be consistent with the terminology usually used in the context of GraphQL.